app/uk/gov/hmrc/testuser/services/TestUserService.scala

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
This file contains a `throw new UserNotFound(userType)` line that has an uncessary new as the UNF is a case class.

Reply:
Changed